src/containers/CountdownManager/CountdownManager.js

```diff
@@ -0,0 +1,45 @@
+import { Component, PropTypes } from 'react';
+
+export default class CountdownManager extends Component {
```

Comment:
Should be renamed to `<Countdown />` and then the normal component should probably live inside `<GameLobby />`, until we come across a use-case where it should live inside the components folder.